tests/frame/join_test.py

```diff
@@ -11,16 +11,20 @@
 from tests.utils import compare_dicts
 
 
-def test_inner_join(constructor_with_lazy: Any) -> None:
+def test_inner_join_two_keys(constructor: Any) -> None:
```

Comment:
the paths for the lazy and the non-lazy methods are usually identical...if coverage is still at 100 without `constructor_with_lazy`, I think we don't need it